network-api/networkapi/templates/pages/base.html

```diff
@@ -65,7 +65,9 @@
       </div>
       {% block hero_guts %}{% endblock %}
 
+      <div class="main-content">
```

Comment:
Why a `<div>` instead of a `<main>`? Unless this isn't actually the main content, in which case we probably want to steer clear of using the word `main` in the class name.

Reply:
Ah I asked in the [original ticket](https://github.com/mozilla/foundation.mozilla.org/issues/3925#issuecomment-557756135) about using a class on `<div>` vs `<main>` and it seemed like both [Youri](https://github.com/mozilla/foundation.mozilla.org/issues/3925#issuecomment-558228965) and [you](https://github.com/mozilla/foundation.mozilla.org/issues/3925#issuecomment-558881492) agreed that we should go with the former case?

Reply:
I recall the issue there being we had `<main>` somewhere else, and it would have conflicted, but in that case let's not call this `main-content` but just `content`, especially if the block itself uses that name too.